utils/retriever/src/retriever/mod.rs

```diff
@@ -518,14 +516,23 @@ fn retrieve_from_network<N: Network>(
     ))
 }
 
-fn fetch_from_network(endpoint: &String, program: &String) -> Result<String, UtilError> {
-    let url = format!("{}/program/{}", endpoint, program);
-    let response = ureq::get(&url.clone())
+// Fetch the given endpoint url and return the sanitized response.
+pub fn fetch_from_network(url: &str) -> Result<String, UtilError> {
+    let response = ureq::get(url)
+        .set(&format!("X-Aleo-Leo-{}", env!("CARGO_PKG_VERSION")), "true")
         .call()
         .map_err(|err| UtilError::failed_to_retrieve_from_endpoint(err, Default::default()))?;
     if response.status() == 200 {
-        Ok(response.into_string().unwrap())
+        Ok(response.into_string().unwrap().replace("\\n", "\n").replace('\"', ""))
```

Comment:
Did the format of the response change, such that we needed to make a change here?

Reply:
We were previously doing this in `query` implementation of network retrievals